src/main/java/org/helpberkeley/memberdata/HBParser.java

```diff
@@ -725,11 +691,8 @@ static RestaurantTemplatePost restaurantTemplatePost(final String rawPost) {
     static UploadFile parseFileFromPost(String rawPost) {
 
         for (String line : rawPost.split("\n")) {
-            if (line.contains(Constants.UPLOAD_URI_PREFIX)) {
-                String shortURL =  shortURLDiscoursePost(line);
-                String fileName = downloadFileName(line);
-
-                return new UploadFile(fileName, shortURL);
+            if (line.contains(Constants.UPLOAD_URI_PREFIX) || line.contains(Constants.WEB_CSV_PREFIX)) {
```

Comment:
Use UploadFile.containsUploadedFileURL();
This is how we hide the details of what this URL looks like.